packages/checkout-extension/src/handlers/handlers.test.ts

```diff
@@ -34,9 +38,6 @@ describe('Handlers', () => {
 
             handler.handler({
                 type: ExtensionCommandType.ReloadCheckout,
-                payload: {
-                    extensionId: '123',
-                },
```

Comment:
This is incorrect. `extensionId` optionally appears in `context` of every message, not in `payload`.
```
context: {
                    extensionId: '123',
                },
```